### ISSUE TITLE
Error when using ticket comments on the proxy version

### ISSUE THREAD

**Issue by RodneyMKay** [HoshiKurama/TicketManager#26]
Getting the following error when someone comments on a ticket while using the proxy connected discord notifications:

```
java.lang.IllegalArgumentException: No enum constant com.github.hoshikurama.ticketmanager.common.discord.DiscordTarget.Type.<playername>
        at java.lang.Enum.valueOf(Enum.java:273) ~[?:?]
        at com.github.hoshikurama.ticketmanager.common.discord.DiscordTarget$Type.valueOf(UserTargets.kt) ~[?:?]
        at com.github.hoshikurama.ticketmanager.common.discord.UserTargetsKt.decodeToDiscordTarget(UserTargets.kt:33) ~[?:?]
        at com.github.hoshikurama.ticketmanager.common.discord.notifications.DiscordNotification$Companion.decode$computeUser(DiscordNotification.kt:23) ~[?:?]
        at com.github.hoshikurama.ticketmanager.common.discord.notifications.DiscordNotification$Companion.decode(DiscordNotification.kt:42) ~[?:?]
        at com.github.hoshikurama.ticketmanager.waterfall.WaterfallBridge.onMessage(WaterfallBridge.kt:166) ~[?:?]
        at jdk.internal.reflect.GeneratedMethodAccessor44.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at net.md_5.bungee.event.EventHandlerMethod.invoke(EventHandlerMethod.java:19) ~[server.jar:git:Waterfall-Bootstrap:1.
        .......
```

Issue seems to be that you `user.name` instead of `user.toString()` here:
https://github.com/HoshiKurama/TicketManager/blob/c7878f31e40b5a246fa7f5c2b0add28c9b40d988/common/src/main/kotlin/com/github/hoshikurama/ticketmanager/common/discord/notifications/Comment.kt#L14

**Comment by HoshiKurama** [comment]
Hey! I apologize for not responding sooner. I will look at this right now.

**Comment by HoshiKurama** [comment]
Fixed with 9.0.1!
